plugins/core/hu.bme.mit.gamma.ui/src/hu/bme/mit/gamma/ui/taskhandler/VerificationHandler.java

```diff
@@ -131,6 +135,7 @@ public VerificationHandler(IFile file, boolean serializeTraces) {
 	//
 	
 	public void execute(Verification verification) throws IOException, InterruptedException {
+		this.verification = verification;
```

Comment:
Is this really necessary? Isn't the `programmingLanguage` the only kind of data you need in an invoked method? It could be a parameter...